_config.yml

```diff
@@ -33,8 +32,6 @@ source:      .
 destination: ./_site
 
 timezone: Asia/Calcutta
-
-devurl: http://localhost:4000
-port: 4000
-host: 0.0.0.0
-baseurl: https://iamit.in/
+short_url: 'iamit.in/'
+production_url:  https://aktech.github.io/
+source_url:      https://github.com/aktech`.github.io
```

Comment:
there are extra spaces after `production_url ` and `source_url `
and and extra backtick after `aktech`

Reply:
Fixed the typo and formating.